charts/StackedBarChart.tsx

```diff
@@ -418,6 +419,18 @@ export class StackedBarChart extends React.Component<{
         const { stackedData, xValues } = this.transform
         const { innerBounds } = axisBox
 
+        const textColor = "#666"
+
+        const tickMarks = true ? (
```

Comment:
This conditional is always true? Is it accidentally here or intentionally?

Reply:
Accident! Thanks.

Making a minor note to myself to look into adding the no-unnecessary-condition rule to our tslint "https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unnecessary-condition.md"